src/modules/previewpane/MonacoPreviewHandler/Settings.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Drawing;
+using System.IO;
+using System.Reflection;
+using Microsoft.PowerToys.PreviewHandler.Monaco.Properties;
+using WK.Libraries.WTL;
```

Comment:
what is this?

Reply:
The themelistener class. I will look how we can move away from it